sqldelight-gradle-plugin/src/test/integration-postgresql/src/test/kotlin/app/cash/sqldelight/postgresql/integration/PostgreSqlTest.kt

```diff
@@ -252,12 +252,31 @@ class PostgreSqlTest {
     assertThat(now).isGreaterThan(OffsetDateTime.MIN)
   }
 
+  @Test fun nowPlusInterval() {
```

Comment:
Minor, you could also use `SELECT NOW(), NOW() + INTERVAL '1 day'` and compare the two values in Java/Kotlin.

Reply:
Ok - have compared now plus 1 > now